src/api/db/schemas/Task.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+import MongoPaging from 'mongo-cursor-pagination';
+
+const Schema = mongoose.Schema;
+
+const TaskSchema = new Schema({
+  user: { type: String, required: true },
+  projectId: { type: String, required: true, ref: 'Project' },
+  type: {
+    type: String,
+    required: true,
+    enum : ['GetStats']
+  },
+  status: {
+    type: String,
+    required: true,
+    enum : ['SUBMITTED', 'RUNNING', 'FAIL', 'COMPLETE'],
+    default: 'SUBMITTED'
+  },
+  created: { type: Date, default: Date.now, required: true },
+  updated: { type: Date, default: Date.now, required: true },
+  output: { type: Object }
```

Comment:
@ingalls - In the case of `getStats`, the complete output will be a JSON object, but this may not be the case for other async tasks (e.g. `export`'s output gets written to S3, so we return a URL to it to the frontend, other async tasks like updating deployments will probably not have an output unless maybe something fails and we decide we want to return an message).

Anyhow I'm wondering what your thoughts are on how we'll handle those cases?

Reply:
As discussed on Slack, we'll just return `output: { url: "s3 string" }` in the case of tasks that produce large files.